chaoscenter/authentication/api/handlers/rest/project_handler.go

```diff
@@ -186,6 +186,15 @@ func GetActiveProjectMembers(service services.ApplicationService) gin.HandlerFun
 	return func(c *gin.Context) {
 		projectID := c.Param("project_id")
 		state := c.Param("state")
+		err := validations.RbacValidator(c.MustGet("uid").(string), projectID,
```

Comment:
Can you please add one more check here, if the user has admin role, then user can view the information.